benchmarks/en-nzema/jw300-baseline/Salomey_eng-nzema.ipynb

```diff
@@ -0,0 +1,725 @@
+{
```

Comment:
<p>Based on the above, it looks like you haven't updated it to train for <code>nzi</code>! This is training one for <code>xh</code>   i.e. isiXhosa</p><p> </p><p>Is this the correct notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane-mt/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I think i lost the main notebook...I don't know how but am still searching for it

Reply:
The notebook is wrong
Just confirmed. If i can't find it then i need to retrain